### PR TITLE
Fix mob status crash & dmg from map hazard

### PR DESCRIPTION
**Pull request by ronancpl**
* Supress some mob status to prevent crashing players on change maps.
* Show player dmg from map hazard to other players.

### CODE REVIEW

File: src/main/java/tools/MaplePacketCreator.java [hunk 1]
```diff
@@ -1439,7 +1439,12 @@ private static void encodeParentlessMobSpawnEffect(MaplePacketLittleEndianWriter
         
         private static void encodeTemporary(MaplePacketLittleEndianWriter mplew, Map<MonsterStatus, MonsterStatusEffect> stati) {
                 int pCounter = -1, mCounter = -1;
-            
+
+                Map<MonsterStatus, MonsterStatusEffect> stati = life.getStati().entrySet()  // to patch some status crashing players
```
Comment (P0nk) [inline src/main/java/tools/MaplePacketCreator.java:1443]: Doesn't compile. No field "life" here.

File: src/main/java/tools/MaplePacketCreator.java [hunk 2]
```diff
@@ -1439,7 +1439,12 @@ private static void encodeParentlessMobSpawnEffect(MaplePacketLittleEndianWriter
         
         private static void encodeTemporary(MaplePacketLittleEndianWriter mplew, Map<MonsterStatus, MonsterStatusEffect> stati) {
                 int pCounter = -1, mCounter = -1;
-            
+
+                stati = stati.entrySet()  // to patch some status crashing players
+                        .stream()
+                        .filter(e -> !(e.getKey().equals(MonsterStatus.WATK) || e.getKey().equals(MonsterStatus.WDEF)))
+                        .collect(Collectors.toMap(e -> e.getKey(), e -> e.getValue()));
```
Comment (P0nk) [inline src/main/java/tools/MaplePacketCreator.java:1446]: `Collectors` is not imported, so this does not compile. You should get into the habit of actually running and testing your code before submitting it in a PR.

I fixed the import and tested this myself, and it looks good. Previously the hazard damage displayed as "Miss", and now it shows the damage. Great fix, thank you.

**Review by P0nk — CHANGES_REQUESTED** [review]

File: src/main/java/tools/MaplePacketCreator.java (hunk 2 above)
Comment (ronancpl) [reply src/main/java/tools/MaplePacketCreator.java:1446]: I admit I did not compile the code within Cosmic's proposed stacks, but the code have been tested and run proper before proposing this PR. Will take into consideration finer reviews at next code exchanges.
Glad it worked!

File: src/main/java/tools/MaplePacketCreator.java [hunk 3]
```diff
@@ -70,9 +70,9 @@
 import java.net.InetAddress;
 import java.sql.ResultSet;
 import java.sql.SQLException;
-import java.util.List;
```
Comment (P0nk) [inline src/main/java/tools/MaplePacketCreator.java:73]: Why did you remove java.util.List? Now we get this error instead:
`java: reference to List is ambiguous
  both interface java.util.List in java.util and class java.awt.List in java.awt match`

**Review by P0nk — APPROVED** [review]
